modules/UI/util/MessageHandler.js

```diff
@@ -108,7 +108,7 @@ var messageHandler = {
             persistent: false,
             buttons: buttons,
             defaultButton: defaultButton,
-            focus: focus,
+            focus: (focus) ? focus : '',
```

Comment:
Why do we need this?

According to the documentation the value should be "Index of the button to focus or selector to element to focus. Default: 0". Am I missing something? Maybe I'm not looking into correct documentation.